File: gems/decomplex/examples/swift/function-lcom.swift
func mixed(price: Int, tax: Int, logger: Logger) -> Result {
  let subtotal = price + tax
  let total = subtotal * 2
  let rounded = total.round()

  let timestamp = now()
  let buffer = Buffer.init()
  buffer.push(timestamp)
  logger.info(buffer)

  return Result.init(rounded, buffer)
}

File: gems/decomplex/examples/php/function-lcom.php
<?php
function mixed($price, $tax, $logger) {
  $subtotal = $price + $tax;
  $total = $subtotal * 2;
  $rounded = $total->round();

  $timestamp = now();
  $buffer = Buffer::init();
  $buffer->push($timestamp);
  $logger->info($buffer);

  return Result::init($rounded, $buffer);
}
